pandas/_libs/tslibs/conversion.pyx

```diff
@@ -74,7 +74,9 @@ cdef inline int64_t get_datetime64_nanos(object val) except? -1:
     return ival
 
 
-def ensure_datetime64ns(ndarray arr, copy=True):
+@cython.boundscheck(False)
+@cython.wraparound(False)
+def ensure_datetime64ns(arr: ndarray, copy: bint = True):
```

Comment:
no spaces on the args

Reply:
I've been trying to figure this out too.  Are you sure about this?  I've been following the usage here https://www.python.org/dev/peps/pep-3107/#syntax